internal/ir/xds.go

```diff
@@ -84,6 +84,8 @@ type Xds struct {
 	UDP []*UDPListener `json:"udp,omitempty" yaml:"udp,omitempty"`
 	// EnvoyPatchPolicies is the intermediate representation of the EnvoyPatchPolicy resource
 	EnvoyPatchPolicies []*EnvoyPatchPolicy `json:"envoyPatchPolicies,omitempty" yaml:"envoyPatchPolicies,omitempty"`
+	// FilterOrder holds the custom order of the HTTP filters
```

Comment:
should this be part of the Translator context instead ?

Reply:
referring to here https://github.com/envoyproxy/gateway/blob/9a626f262317b90946219d7f85069221cce2c4b6/internal/xds/translator/translator.go#L43

Reply:
oh nvm I see both